src/patterns-hmrc-govuk-team/government-gateway/index.njk

```diff
@@ -11,16 +11,14 @@ layout: twoColumnPage.njk
   Use this pattern when you're linking to a service that requires the user to sign in using Government Gateway.
 </p>
 
-<h2 class="govuk-heading-l">Where to place</h2>
+<h2 class="govuk-heading-l">How to design this pattern</h2>
 
-<p class="govuk-body">You should place this pattern before the link to the service, unless there's a <a href="/patterns-hmrc-govuk-team/" class="govuk-link">design pattern</a> that says differently.</p>
+<p class="govuk-body">You should place this pattern before the link to the service, unless there's a <a href="/patterns-hmrc-govuk-team/" class="govuk-link">content pattern</a> that says differently.</p>
 
-<h2 class="govuk-heading-l">How it works</h2>
-
-<h3 class="govuk-heading-m">Option 1</h3>
+<h3 class="govuk-heading-m">For most services</h3>
 
 <p class="govuk-body">
-  Use this option for most services.
+  Use this for most services.
```

Comment:
Should this be a colon at the end? Feel like it wouldn't give a great experience for a screen reader user

Reply:
Done.